scripts/cnode-helper-scripts/cntools.sh

```diff
@@ -1702,6 +1704,53 @@ function main {
               [[ -n ${pool_id_bech32} ]] && println "ID (bech32) : ${pool_id_bech32}"
               waitForInput && continue
               ;; ###################################################################
+           import)
```

Comment:
We might want to limit this to Ledger for now. I couldn't get Trezor to work for pool cold keys. Could be a problem with my Trezor and VM env but I don't think so. 

```bash
cardano-hw-cli node key-gen --path 1853H/1815H/0H/0H --hw-signing-file /opt/cardano/guild/priv/pool/Trezor/cold.hwsfile --cold-verification-key-file /opt/cardano/guild/priv/pool/Trezor/cold.vkey --operational-certificate-issue-counter-file /opt/cardano/guild/priv/pool/Trezor/cold.counter
Error: Extended public key export cancelled by user
```

Reply:
Have you succeeded in having some output with basic "device" command if it exists with TREZOR ?